storage/clickhousespanstore/writer_test.go

```diff
@@ -0,0 +1,308 @@
+package clickhousespanstore
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"sort"
+	"strconv"
+	"testing"
+	"time"
+
+	"gotest.tools/assert"
+
+	"github.com/hashicorp/go-hclog"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/gogo/protobuf/proto"
+	"github.com/jaegertracing/jaeger/model"
+
+	"github.com/pavolloffay/jaeger-clickhouse/storage/clickhousespanstore/mocks"
+)
+
+const (
+	testSpanCount  = 100
+	testTagCount   = 100
+	testIndexTable = "test_index_table"
+	testSpansTable = "test_spans_table"
+)
+
+func TestSpanWriter_TagString(t *testing.T) {
+	tags := generateRandomTags()
```

Comment:
Here it might be better to define tags in with all types as a table and test that directly.

Reply:
https://dave.cheney.net/2019/05/07/prefer-table-driven-tests